magister-calendar.js

```diff
@@ -514,8 +533,16 @@ function calendarItem(action, appointment, googleconfig, retry) {
     form.end.dateTime = new Date(endtime).toISOString();
   }
 
+
+  for(c = 0; c < CONFIG.location_means_cancelled.length; c++) {
+    // If the location means the appointment was cancelled.
+    if (appointment.location == CONFIG.location_means_cancelled[c]) {
+      var location_means_cancelled = true;
+    }
+  }
+
```

Comment:
I wonder if this can be simplified. Your PR adds two `for` loops and they both essentially do the exact same thing.